csharp/TypeDB.Driver.Pinvoke.nuspec

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$packageid$</id>
+    <version>$version$</version>
+    <authors>TypeDB</authors>
+    <copyright>Copyright © 2024 Vaticle</copyright>
+    <owners>TypeDB</owners>
+    <title>TypeDB Driver Runtime</title>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Runtime part of the library for TypeDB C# support.</summary>
+    <description>
+      TypeDB is a polymorphic database with a conceptual data model, a strong subtyping system, a symbolic reasoning engine, and a beautiful and elegant type-theoretic language: TypeQL.
+      This package contains a runtime library for the TypeDB driver: https://www.nuget.org/packages/TypeDB.Driver.
+      Check all the available platform-dependent runtime libraries here: https://www.nuget.org/packages?q=TypeDB.Driver.Pinvoke.
```

Comment:
Is it convention to call this package `TypeDB.Driver.Pinvoke` or is that something you chose? In Java I think `jni` is a pretty standard convention

Reply:
It is not a convention. I thought about naming it a `Runtime`, but:
1. If a user receives a runtime error because of the missing `TypeDB.Driver.Pinvoke` dll, it's easier for them to google nuget `TypeDB.Driver.Pinvoke` and find it rather than come up with an idea of `TypeDB.Driver.Runtime`.
2. Calling a dll `Runtime`, hence the whole namespace `Runtime` was a bit strange for me, but I'm ready to change it if you want me to.

I believe that `Pinvoke` is okay as many people should know about `pinvoke` and its connection with the native libs, and all the packages are called `osx-x64` etc, which also shows platform-specific and runtime nature of the package.

Reply:
Ok let's stick with it then!